helm/reana/README.md

```diff
@@ -18,6 +18,7 @@ This Helm automatically prefixes all names using the release name to avoid colli
 | `components.reana_server.environment.REANA_SCHEDULER_REQUEUE_SLEEP` | Seconds to wait between consuming workflows. | 15 |
 | `components.reana_server.environment.REANA_SCHEDULER_REQUEUE_COUNT` | The number of times to requeue workflow before failing it. | 200 |
 | `components.reana_server.environment.REANA_WORKFLOW_SCHEDULING_POLICY` | Define workflow scheduling strategy. Options are "fifo" for first-in-first-out strategy regardless of users and "balanced" for multi-user-aware scheduling strategy. | "fifo" |
+| `components.reana_server.environment.REANA_WORKFLOW_SCHEDULING_READINESS_CHECK` | Define the checks needed to assess whether the cluster can start new workflows. Options are 0 = switch all the checks off, 1 = only check if upper limit on concurrent REANA batch workflows was not exceeded, 2 = only check if there is enough memory available to schedule the workflow, 9 = all the checks turned on | 9 |
```

Comment:
Text suggestion (although it may be a bit longish!):

-  Define checks that are performed to asses whether the cluster is ready to start new workflows. Values are: 0 = no readiness check; schedule new workflow as soon as they arrive; 1 = check for maximum number of concurrently running workflows; schedule new workflows if not exceeded; 2 = check for available cluster memory size; schedule new workflow only if it fits; 9 = perform all checks; satisfy all previous criteria.